airflow/windows_extensions/Signals.py

```diff
@@ -0,0 +1,35 @@
+import enum
+
+class Signals(enum.Enum):
```

Comment:
Why do need this on Windows? -- it doesn't support most of these signals does it?

Reply:
True, this was basically just to satisfy some of the scripts in airflow.utils. We tried to approximate the behavior of the various signal method by using things like e.g. threads for timers. Certainly not ideal, I am very open to better options.